packages/file/test-files/test.js

```diff
@@ -0,0 +1,6 @@
+const allo = "allo";
```

Comment:
What is this file for ?

Reply:
To test that we can remove stuff inside comments over multiple lines